tardis/apps/publication_workflow/components/PublicationsHome.jsx

```diff
@@ -0,0 +1,103 @@
+import React, {
+  Fragment, useEffect, useState,
+} from 'react';
+import PublicationButton from './PublicationButton';
+import PublicationsList from './PublicationsList';
+import FormModal from './FormModal';
+import { fetchPubs, SubmitFormData } from './utils/FetchData';
+import PublicationToast from './utils/PublicationToast';
+
+const PublicationsHome = () => {
+  const [show, setShow] = useState(false);
+  const [releasedPubsList, setReleasePubsList] = useState([]);
+  const [draftPubsList, setDraftPubsList] = useState([]);
+  const [retractedPubsList, setRetractedPubsList] = useState([]);
+  const [scheduledPubsList, setScheduledPubsList] = useState([]);
+  const [resumeDraftId, setResumeDraftId] = useState(0);
+  const [initialData, setInitialData] = useState({});
+  const [toastShow, setToastShow] = useState(false);
+  const [toastMessage, setToastMessage] = useState('');
+  const [listViewType, setListViewType] = useState('grid');
+  const onResumeDraft = (e, id) => {
+    setResumeDraftId(id);
+    // fetch  data
+    SubmitFormData({}, 'resume', id).then((data) => {
+      setInitialData(data);
+      setShow(true);
+    });
+  };
+  const handleShow = () => setShow(true);
+
+  const onPubUpdate = () => {
+    // TODO only load specific pubType
+    fetchPubs('draft').then((data) => {
+      setDraftPubsList(data);
+    });
+    fetchPubs('released').then((data) => {
+      setReleasePubsList(data);
+    });
+    fetchPubs('retracted').then((data) => {
+      setRetractedPubsList(data);
+    });
+    fetchPubs('scheduled').then((data) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
    fetchPubs("scheduled").then((data) => {
```